### ISSUE TITLE
Only first item in a list shows ripple effect

### ISSUE THREAD

**Issue by divanshugarg** [ZieIony/Carbon#76]
On my Android 5.0.2 (API 21), in a list view with ripple effect enabled on items. Only the first item shows the ripple effect, others dont. This problem happens in the samples too.

**Comment by gabber12** [comment]
This happened to me too but using carbon.widget.LinearLayout somehow fixed this.

**Comment by ZieIony** [comment]
Could you please provide more information about the device you're testing on and the screen (from samples) you noticed the issue?

**Comment by ZieIony** [comment]
ListView processes clicks in a different way in order to properly draw selectors under right items. Setting a RippleDrawableCompat instance as a selector doesn't work, because there was no setHotspot method before Lollipop. Setting such drawable as an item's background doesn't work, because of that special clicks processing.

See:

http://stackoverflow.com/questions/28072817/ripple-effect-only-working-on-first-listview-item?lq=1

http://www.reddit.com/r/androiddev/comments/2t7cd6/help_ripple_effect_only_working_on_first_listview/

To sum up - there's no good way of using animated drawables as selectors with ListView. That's why ListView in Carbon 0.8.0 is marked as depreciated. You should use RecyclerView and specify a ripple color on your item's root layout. I'm going to update the sample app to reflect that change.
